### ISSUE TITLE
Notice(8): Undefined index: virtualmachine_creation

### ISSUE THREAD

**Issue by ddurieux** [fusioninventory/fusioninventory-for-glpi#1584]
---

Author Name: **Meg Skywalker** (Meg Skywalker)
Original Redmine Issue: 2836, http://forge.fusioninventory.org/issues/2836
Original Date: 2014-12-19

---

I got these errors every 4 hours. Seems a cron job.

GLPI 0.84.8

```
2014-12-19 08:30:36
Notice(8): Undefined index: virtualmachine_creation
Backtrace :
/usr/share/glpi/plugins/fusioninventory/inc/inventorycomputerlib.class.php:887          Toolbox::userErrorHandlerNormal()
/usr/share/glpi/plugins/fusioninventory/inc/inventorycomputerinventory.class.php:532            PluginFusioninventoryInventoryComputerLib->updateComputer()
/usr/share/glpi/plugins/fusioninventory/inc/inventoryruleimport.class.php:664           PluginFusioninventoryInventoryComputerInventory->rulepassed()
/usr/share/glpi/inc/rule.class.php:997          PluginFusioninventoryInventoryRuleImport->executeActions()
/usr/share/glpi/inc/rulecollection.class.php:695                Rule->process()
/usr/share/glpi/plugins/fusioninventory/inc/inventorycomputerinventory.class.php:350            RuleCollection->processAllRules()
/usr/share/glpi/plugins/fusioninventory/inc/inventorycomputerinventory.class.php:104            PluginFusioninventoryInventoryComputerInventory->sendCriteria()
/usr/share/glpi/plugins/fusioninventory/inc/communication.class.php:222         PluginFusioninventoryInventoryComputerInventory->import()
/usr/share/glpi/plugins/fusioninventory/inc/communication.class.php:438         PluginFusioninventoryCommunication->import()
/usr/share/glpi/plugins/fusioninventory/front/communication.php:85              PluginFusioninventoryCommunication->handleOCSCommunication()
/usr/share/glpi/plugins/fusioninventory/index.php:50            include_once()

2014-12-19 08:30:36
Warning(2): Invalid argument supplied for foreach()
Backtrace :
/usr/share/glpi/plugins/fusioninventory/inc/inventorycomputerlib.class.php:887          Toolbox::userErrorHandlerNormal()
/usr/share/glpi/plugins/fusioninventory/inc/inventorycomputerinventory.class.php:532            PluginFusioninventoryInventoryComputerLib->updateComputer()
/usr/share/glpi/plugins/fusioninventory/inc/inventoryruleimport.class.php:664           PluginFusioninventoryInventoryComputerInventory->rulepassed()
/usr/share/glpi/inc/rule.class.php:997          PluginFusioninventoryInventoryRuleImport->executeActions()
/usr/share/glpi/inc/rulecollection.class.php:695                Rule->process()
/usr/share/glpi/plugins/fusioninventory/inc/inventorycomputerinventory.class.php:350            RuleCollection->processAllRules()
/usr/share/glpi/plugins/fusioninventory/inc/inventorycomputerinventory.class.php:104            PluginFusioninventoryInventoryComputerInventory->sendCriteria()
/usr/share/glpi/plugins/fusioninventory/inc/communication.class.php:222         PluginFusioninventoryInventoryComputerInventory->import()
/usr/share/glpi/plugins/fusioninventory/inc/communication.class.php:438         PluginFusioninventoryCommunication->import()
/usr/share/glpi/plugins/fusioninventory/front/communication.php:85              PluginFusioninventoryCommunication->handleOCSCommunication()
/usr/share/glpi/plugins/fusioninventory/index.php:50            include_once()

```

**Comment by ddurieux** [comment]
---

Original Redmine Comment
Author Name: **David Durieux** (@ddurieux)
Original Date: 2014-12-24T08:05:58Z

---

It's not a cron, it's a computer inventory

**Comment by ddurieux** [comment]
Yet fixed in previous commit (tested in 0.90.2 version)
